core/scheduler/service/job_lineage_service.go

```diff
@@ -43,9 +43,13 @@ func (j *JobLineageService) GetJobExecutionSummary(ctx context.Context, jobSched
 	return result, nil
 }
 
-func (j *JobLineageService) GetJobLineage(ctx context.Context, jobSchedules []*scheduler.JobSchedule) (map[scheduler.JobName]*scheduler.JobLineageSummary, error) {
+func (j *JobLineageService) GetJobLineage(ctx context.Context, jobSchedules map[scheduler.JobName]*scheduler.JobSchedule) (map[scheduler.JobName]*scheduler.JobLineageSummary, error) {
```

Comment:
how do we handle for hourly job here, if it isa map

Reply:
currently we're not handling hourly job properly, we can do that later